src/modules/SX126x/SX126x.cpp

```diff
@@ -1345,6 +1354,19 @@ int16_t SX126x::setPacketMode(uint8_t mode, uint8_t len) {
   return(state);
 }
 
+int16_t SX126x::setHeaderType(uint8_t headerType) {
+  // set requested packet mode
+  int16_t state = setPacketParams(_preambleLength, _crcType, 0xFF, headerType);
```

Comment:
I hope using 0xFF here is the right thing to do, as it will presumably be overridden before TX/RX operations